library/README.md

```diff
@@ -0,0 +1,83 @@
+# Swarm Browser Extension JS Library
+
+Library for interaction with the Swarm Browser Extension, from Dapps or other browser extensions.
+
+## Installation
+
+The library can be installed via npm:
+
+```bash
+npm install --save @ethersphere/swarm-extension
+```
+
+## Usage
+
+### Swarm class
+
+All interaction with the Swarm browser extension is established through the Swarm class:
+
+```typescript
+import { Swarm } from '@fairdatasociety/swarm-extension'
+```
+
+By default the class will connect to the Swarm browser extension using its ID from the Google store. If you
+are running your version of the extension the class can be configured with a different extension ID.
+
+```typescript
+const swarm = new Swarm() // Using the default Swarm ID from the Google store
+```
+
+```typescript
+const swarm = new Swarm('Swarm Extension ID...') // Using custom Swarm ID
+```
+
+### Swarm class functionalities
+
+Before interacting with the library, Dapp should register new session by calling:
+
+```typescript
+await swarm.register()
```

Comment:
this should happen automatically at some point (at the start of swarm-html or - if that was not imported then - at the library import)